content/de/docs/faq/_index.md

```diff
@@ -0,0 +1,624 @@
+---
```

Comment:
We have refactored the FAQ to be several pages instead of one. I will merge this, but I suggest you take a look at those updates in https://github.com/helm/helm-www/tree/main/content/en/docs/faq so that you can make the german as updated as possible.